packages/react-app-revamp/components/_pages/WizardFormCreateContest/Step3/index.tsx

```diff
@@ -232,7 +232,7 @@ export const Step3 = () => {
                       chain: contestDeployedToChain?.name.toLowerCase().replace(" ", ""),
                       address: dataDeployContest?.address,
                       tokenRewardsAddress: dataContestRewardsModule.tokenRewardsAddress,
-                      totalRewards: dataContestRewardsModule.rewardsTotalAmount,
```

Comment:
Should we just remove this?

Reply:
Keeping `totalRewards` here with a default value of `0` so that when the user clicks on the `Add funds to your contest rewards module` link and open the "Fund rewards pool" modal, the value in the dialog is initialized (if `totalRewards` was equal to 20 and `tokenRewardsAddress` would be the address of DAI, then the form would display 20 DAI by default). I suggest we keep this behaviour for now since the flow for how/when funding the reward pool is still being worked on.

Reply:
Gotcha, thank you for laying this out, sounds good!